readme.md

```diff
@@ -2,8 +2,8 @@
 
 | VIC Links & Badges              |                                                                             |
 |------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| VIC Documentation      | [![Documentation Status](https://readthedocs.org/projects/vic/badge/?version=latest)](http://vic.readthedocs.org/en/latest/)                                                                             |
-| Travis Build           | [![Build Status](https://travis-ci.org/UW-Hydro/VIC.png)](https://travis-ci.org/UW-Hydro/VIC)                                                                                             |
+| VIC Documentation      | [![Documentation Status](https://readthedocs.org/projects/vic/badge/?version=latest]](http://vic.readthedocs.org/en/latest/)                                                                             |
```

Comment:
Did you mean to make this change?

Reply:
yes, it was out of place and I was just be particular.

Reply:
It's actually doesn't change anything on the rendered markdown.